__tests__/EditHistoryTitleCollapsedActions.test.js

```diff
@@ -0,0 +1,39 @@
+jest.mock('app/services/Firebase', () => {});
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+
+import * as Analytics from 'app/services/Analytics';
```

Comment:
- [ ] Not on this file but test `watch_video`'s new property `from_collapsed_card`

Reply:
skipping